test/com/amazon/ion/streaming/ReaderIntegerSizeTest.java

```diff
@@ -142,9 +142,11 @@ public void testGetIntegerSizeNegativeIntBoundary()
     private void testGetIntegerSizeIntBoundary(int boundaryValue, long pastBoundary)
     {
         in.next();
-        assertEquals(IntegerSize.INT, in.getIntegerSize());
+        // It's fine if IntegerSize recommends a larger-than-necessary type, just not a smaller one.
+        assertTrue(IntegerSize.INT == in.getIntegerSize() || IntegerSize.LONG == in.getIntegerSize());
```

Comment:
It repeats twice (both in this method). The same pattern occurs in `testGetIntegerSizeLongBoundary`, but with `LONG` and `BIG_INTEGER` instead of `INT` and `LONG`. In this case I'm okay without a method, as it wouldn't reduce the amount of code or make much (if any) positive impact on readability.